pkg/printers/internalversion/printers.go

```diff
@@ -1673,11 +1673,24 @@ func printEvent(obj *api.Event, options printers.GenerateOptions) ([]metav1.Tabl
 	if obj.FirstTimestamp.IsZero() {
 		firstTimestamp = translateMicroTimestampSince(obj.EventTime)
 	}
+
 	lastTimestamp := translateTimestampSince(obj.LastTimestamp)
 	if obj.LastTimestamp.IsZero() {
 		lastTimestamp = firstTimestamp
 	}
 
+	count := obj.Count
+	if obj.Series != nil {
+		lastTimestamp = translateMicroTimestampSince(obj.Series.LastObservedTime)
+		// Add the first event to the count.
+		count = obj.Series.Count + 1
+	}
+
+	// Singleton events don't have a count set in the new API.
+	if count == 0 {
```

Comment:
This actually could be and else to the above condition, since a singleton will not have `Series` set either.